feature-proposals.rst

```diff
@@ -30,111 +36,175 @@ a new Wiki page containing an RFC with a unique title MUST be created. In this
+level of activity. Any message to the discussion thread from any participant 
+(excluding “Intent to Vote” messages) is sufficient to keep the RFC active. Once an RFC discussion
+goes inactive, any new post will "reactivate" the discussion and trigger a new
+cooldown period.
+
+-  If it has been 42 days (~6 weeks) since the last message, the cooldown is 7 days.
+-  If it has been 90 days (~3 months) since the last message, the cooldown is 14 days.
 
 **************
  Voting Phase
 **************
 
-RFC authors MAY start a vote after the cooldown period has elapsed. The
-intention of starting the vote (“voting intent”) MUST be announced at least 2
-days (48 hours) before the start of the vote in the official discussion thread.
+Voting prerequisites
+====================
 
-RFC authors SHOULD NOT announce the voting intent when the RFC discussion is
-still ongoing and SHOULD consider all new discussion points that are brought
-forward. RFC authors SHOULD also consider all new discussion points brought
-forward after announcing the voting intent before actually proceeding with
-starting the vote. If objective improvements are brought forward after
-announcing the voting intent they SHOULD NOT be disregarded in the interest of
-time and at the expense of quality.
+Prior to starting a vote, RFC authors MUST post an Intent to Vote message to
+the discussion thread. The post MUST be made at least 2 days and no more than
+7 days before the vote is officially opened. Any feedback posted
+after the Intent to Vote message is sent SHOULD be treated the same as if it had
+been sent earlier, and any Major or Minor changes that result from that feedback
+MUST trigger a new cooldown period, canceling the Intent to Vote.
 
-If major or minor changes to the RFC text are made after announcing the voting
-intent, the vote MUST NOT be started. Instead the cooldown period MUST be reset
-and a new voting intent MUST be announced when the RFC is ready for voting after
-the new cooldown period has elapsed.
+RFC authors MAY start a vote at any time, provided that all the following conditions hold:
+
+-  There is no cooldown period still active.
+
+-  The discussion thread is not inactive.
+
+-  The authors has posted an intent to open the vote at least 2 days prior, and
+   no more than 7 days prior.
+
+-  There is no ongoing relevant and substantive discussion still happening in
+   the thread. The authors may determine what qualifies as relevant and
+   substantive, but SHOULD be liberal in interpreting that.
+
+All voting widgets on the RFC MUST include all relevant details for that vote, as
+described in the "Required Majority" section below.
+
+To allow for timezone differences, outside commitments by RFC authors, and other deadlines,
+the cutoff timestamp for the cooldown period or to start a vote has a grace period of
+8 hours in either direction. For example, the following scenarios are allowed, but discouraged.
```

Comment:
```suggestion
To account for minor calculation errors due to timezone changes,
outside commitments by RFC authors, and others deadlines, RFC
authors MAY use a grace period of up to 8 hours in either direction
without being in violation of the cooldown period and the Intent
to Vote lifetime. RFC authors SHOULD nevertheless strive to
observe the cut-off timestamps as closely as possible.

As an example, all of the following scenarios would be allowed,
but discouraged:
```

Wanted to include the MAY and SHOULD keywords. And also wanted to explicitly spell out the “minor calculation errors” as the primary purpose of the grace period.